src/praisonai-agents/praisonaiagents/push/client.py

```
@@ -0,0 +1,380 @@
+        while self._connected:
+            try:
+                await asyncio.sleep(15)
+                if self._connected:
+                    await self._send({
+                        "type": "presence.heartbeat",
+                        "status": "online",
+                    })
+            except asyncio.CancelledError:
+                break
+            except Exception:
+                pass
```

Ruff (0.15.10)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)